cmd/translate/main_grpc_integration_test.go

```diff
@@ -423,12 +423,12 @@ func Test_ListServices_gRPC(t *testing.T) {
 
 // ------------------Messages------------------
 
-func randMessages(t *testing.T, override *translatev1.Messages) *translatev1.Messages {
+func randMessages(t *testing.T, langOverride string) *translatev1.Messages {
```

Comment:
why change from `*translatev1.Messages` to `string`?

Reply:
in this `randMessages`, the only purpose of override was to override messages language.
so passing only language is easier than passing the whole message struct, just to get new language

Reply:
@VladislavsPerkanuks did this change and since I had a day off, I don't know the reason why it is changed

Reply:
It's gonna go back to `override *translatev1.Messages` at one point - when `*translatev1.Messages` becomes more complex with more fields.